code/modules/power/supermatter/supermatter.dm

```diff
@@ -755,6 +755,13 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		var/mob/living/user = AM
 		if(user.status_flags & GODMODE)
 			return
+		if(ishuman(AM))
+			var/mob/living/carbon/human/proofed = user
+			if(proofed.dna && SUPERMATTERPROOF in proofed.dna.mutations)
+				if(proofed.client)
+					to_chat(proofed, "<span class='notice'>[src] kinda stings a bit.</span>")
+					proofed.client.give_award(/datum/award/achievement/misc/supermatter_survived, proofed)
+			return
```

Comment:
If it's human it's dust immune?

Reply:
read above

Reply:
jokes on you I deleted above like a moron